### PR TITLE
Update version increment to wrap correctly.

### PR DESCRIPTION
**Pull request by ygottlieb-vencorelabs-com**
Pull request to patch issue 549

### CODE REVIEW

**Comment by antoninbas** [comment]
Please add a unit test (https://github.com/p4lang/behavioral-model/blob/master/tests/test_tables.cpp) that fails without the fix and passes with the fix.

Also confirm that you or your organization is a p4.org member, otherwise I cannot accept the patch.

**Comment by ygottlieb-vencorelabs-com** [comment]
Antonin,

Thank you for your quick response.

I've added a test to test_tables.cpp as requested.

We are not yet P4.org members---our membership form for P4.org is currently making its way through the approval process.

File: tests/test_tables.cpp [hunk 1]
```diff
@@ -354,6 +354,22 @@ TYPED_TEST(TableSizeTwo, DeleteEntryHandleUpdate) {
   ASSERT_EQ(MatchErrorCode::EXPIRED_HANDLE, rc);
 }
```
Comment (antoninbas) [inline tests/test_tables.cpp]: Please add a quick comment here pointing to your Github issue: https://github.com/p4lang/behavioral-model/issues/549

File: tests/test_tables.cpp [hunk 2]
```diff
@@ -354,6 +354,22 @@ TYPED_TEST(TableSizeTwo, DeleteEntryHandleUpdate) {
...
 
+TYPED_TEST(TableSizeTwo, HandleVersionWrapAround) {
+
```
Comment (antoninbas) [inline tests/test_tables.cpp]: remove extra space

File: tests/test_tables.cpp [hunk 3]
```diff
@@ -354,6 +354,22 @@ TYPED_TEST(TableSizeTwo, DeleteEntryHandleUpdate) {
...
+  std::string key_ = "\xaa\xaa";
+  ByteContainer key("0xaaaa");
+  entry_handle_t handle;
+  MatchErrorCode rc;
+
+  for (unsigned int i = 0; i < 257; ++i) {
+
```
Comment (antoninbas) [inline tests/test_tables.cpp]: remove extra space

**Comment by ygottlieb-vencorelabs-com** [comment]
I've removed the spaces and added the note.  I'll let you know when the membership is finalized.

**Comment by antoninbas** [comment]
@ygottlieb-vencorelabs-com any update on this? I'd like to be able to merge the fix since this is a pretty serious issue...

**Comment by ygottlieb-vencorelabs-com** [comment]
@antoninbas the application is still making its way through the appropriate channels, hopefully in the next couple of days.

**Comment by antoninbas** [comment]
@ygottlieb-vencorelabs-com I now see you on the p4.org members webpage.
The Travis CI testing is currently failing because of a style issue (extra whitespace somewhere). Would you mind fixing it so I can proceed with merging your PR?

**Comment by antoninbas** [comment]
Hi @ygottlieb-vencorelabs-com. I really need this fix for the next release. If I don't hear from you this week, I will open a new PR and pull in your commit.

**Comment by antoninbas** [comment]
Thanks @ygottlieb-vencorelabs-com !

**Pull request merged**